### DISCUSSION TITLE
Where is Learn directory in github? Is nextjs.org/learn not public?

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#48532] [Help]
### Summary

I would love to have access to [nextjs.org/learn ](https://nextjs.org/learn/foundations/about-nextjs) directory files, but didn't find on github. Are they public? Where can I get them?

![image](https://user-images.githubusercontent.com/130306508/232771474-6b1eff41-40ac-4326-b4b9-e0f0bedff36a.png)

**Comment by icyJoseph** [comment]
This is as close as you can get IIRC: https://github.com/vercel/next-learn

**Reply by ghost** [reply]
I did check that repo before, but I would love to have full version of [nextjs.org/learn](https://nextjs.org/learn/foundations/about-nextjs).

**Reply by icyJoseph** [reply]
Yeah, as far as I remember it is closed source. If you found some issue, you can report it on that repository.
